src/index.js

```diff
@@ -0,0 +1,84 @@
+import places from 'places.js';
+import Header from './components/Header';
+import Main from './components/Main';
+import { fetchUrl, fetchData } from './components/FetchData';
+import './css/style.css';
+
+
+const page = document.querySelector('#content');
+page.append(Header(), Main());
+
+// start search name autocomplete
+const getSearch = document.querySelector('#search');
+const fixedOptions = {
+  appId: 'plN8796X3W30',
+  apiKey: 'c568c1cb9061fe015472a6d3ddbb9356',
+  container: getSearch,
+};
+places(fixedOptions);
+const getCity = document.querySelector('#cityName');
+const run = async (name) => {
+  const url = fetchUrl(name);
+  const responce = await fetchData(url);
+  return responce;
+};
+
+const reset = () => {
+  getCity.innerHTML = '';
+  document.querySelector('#mainOne h5').innerHTML = '';
+  document.querySelector('#mainTwo h5').innerHTML = '';
+  document.querySelector('#mainThree h5').innerHTML = '';
+  document.querySelector('#mainFour h5').innerHTML = '';
+};
+
+const appendData = async () => {
+  const data = 'Nairobi';
```

Comment:
javascript allows us to avoid this, you can declare  a variable and initialize later